src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -7262,11 +7263,19 @@ subroutine summarize_timestep(domain)
        real (kind=RKIND) :: lonMax, lonMax_global
        real (kind=RKIND), dimension(5) :: localVals, globalVals
 
-       real (kind=RKIND) :: spd
+       real (kind=RKIND), dimension(:,:), allocatable :: spd
+       !$acc declare create(spd)
```

Comment:
Everything looks good, but I had a question about why the declare create is required here. Wasn't fully clear about that from our previous meetings.

Reply:
I did this just because it seems to be what is preferred by the OpenACC standard for Fortran allocatable arrays.

`acc declare create(spd)` allows the compiled code to "just do the right thing" whenever it encounters the `allocate(spd(...))` or `deallocate(spd)` statements later on - in this case, allocate this array on both device and host.

Reply:
Also, since `spd` won't be fused into any data region (and thus isn't a type of temporary data movement) it doesn't need to be tracked in any `MPAS_ACC_TIMER_...` region.

Reply:
Got it. Thanks for the explanation!